docs/en_us/platform_api/Makefile

```diff
@@ -1,164 +0,0 @@
-# Makefile for Sphinx documentation
```

Comment:
@catong This hasn't merged and won't until you complete your work.  So there is nothing to revert.